.github/workflows/docs.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           pip install tox
           tox -e docs-py39
       - name: Upload docs artifact
-        uses: actions/upload-pages-artifact@v3
+        uses: actions/upload-pages-artifact@v1
```

Comment:
I am currently testing to deploy this exact branch on my fork. Will update this comment once I have a result.

Reply:
did you test v2?